src/main/resources/assets/bloodmagic/lang/fr_FR.lang

```diff
@@ -1,239 +1,250 @@
+item.BloodMagic.sentientBow.name=Arc Sentiente
+item.BloodMagic.sentientArmourGem.name=Gemme d'Armure Sentiente
+item.BloodMagic.sentientAxe.name=Hâche Sentiente
+item.BloodMagic.sentientPickaxe.name=Pioche Sentiente
+item.BloodMagic.sentientShovel.name=Pelle Sentiente
+
+item.BloodMagic.nodeRouter.name=Routeur de Jonction
+item.BloodMagic.itemFilter.exact.name=Filtre d'Objets Précis
+item.BloodMagic.itemFilter.ignoreNBT.name=Filtre d'Objets par NBT
+item.BloodMagic.itemFilter.modItems.name=Filtre d'Objets par Mod
+item.BloodMagic.itemFilter.oreDict.name=Filtre d'Objets par le Catalogue de Minerai
+
+item.BloodMagic.experienceTome.name=Tôme de Peritia
+item.BloodMagic.sanguineBook.name=Livre Sanguin
+
+item.BloodMagic.livingPointUpgrade.draftAngelus.name=Draft d'Angelus
+
+# Blocks
+tile.BloodMagic.fluid.lifeEssence.name=Essence de Vie
+
+tile.BloodMagic.stone.ritual.master.name=Pierre Rituelle Maîtresse
+tile.BloodMagic.stone.ritual.imperfect.name=Pierre Rituelle Imparfaite
+
+tile.BloodMagic.altar.name=Autel de Sang
+tile.BloodMagic.alchemyArray.name=&r&fÉtalage d'Alchimie
+
+tile.BloodMagic.rune.blank.name=Rune de Sang
+tile.BloodMagic.rune.speed.name=Rune de Vitesse
+tile.BloodMagic.rune.efficiency.name=Rune d'Efficacité
+tile.BloodMagic.rune.sacrifice.name=Rune de Sacrifice
+tile.BloodMagic.rune.selfsacrifice.name=Rune de l'Autosacrifice
+tile.BloodMagic.rune.displacement.name=Rune de Dislocation
+tile.BloodMagic.rune.capacity.name=Rune de Capacité
+tile.BloodMagic.rune.augcapacity.name=Rune de Capacité Supérieur
+tile.BloodMagic.rune.orb.name=Rune de l'Orbe
+tile.BloodMagic.rune.acceleration.name=Rune d'Accélération
+tile.BloodMagic.rune.charging.name=Rune de Chargement
+
+
+tile.BloodMagic.ritualStone.blank.name=Pierre Rituelle
+tile.BloodMagic.ritualStone.water.name=Pierre Rituelle d'Eau
+tile.BloodMagic.ritualStone.fire.name=Pierre Rituelle du Feu
+tile.BloodMagic.ritualStone.earth.name=Pierre Rituelle de la Terre
+tile.BloodMagic.ritualStone.air.name=Pierre Rituelle de l'Air
+tile.BloodMagic.ritualStone.dusk.name=Pierre Rituelle du Crépuscule
+tile.BloodMagic.ritualStone.dawn.name=Pierre Rituelle de l'Aube
+
+tile.BloodMagic.bloodstonebrick.large.name=Brique Large de Roche-Sang
+tile.BloodMagic.bloodstonebrick.brick.name=Brique de Roche-Sang
+tile.BloodMagic.crystal.large.name=Groupe de Cristaux
+tile.BloodMagic.crystal.brick.name=Bloc de Groupe de Cristaux
+tile.BloodMagic.bloodLight.name=Lumière Sanguine
+tile.BloodMagic.spectralBlock.name=Bloc Spectral
+tile.BloodMagic.phantom.name=Bloc Fantôme
+tile.BloodMagic.incenseAltar.name=Autel Encens
+
+tile.BloodMagic.teleposer.name=Téléposeur
+tile.BloodMagic.soulForge.name=Hellfire Forge
+tile.BloodMagic.alchemyTable.name=Table d'Alchimie
+tile.BloodMagic.demonCrucible.name=Creuset Diabolique
+tile.BloodMagic.demonPylon.name=Pylône Diabolique
+tile.BloodMagic.demonCrystallizer.name=Demon Crystallizer
+
+tile.BloodMagic.masterRouting.name=Master Routing Node
```

Comment:
"Noeud de routage maître" (not sure about "noeud", sounds weird)
for the ones below:
Noeud de routage de sortie
Noeud de routage d'entrée
Noeud de routage